Mobile Test Runner for Jama/Mobile Test Runner for JamaTests/TestCycleListModelUnitTests.swift

```diff
@@ -21,6 +21,7 @@ class TestCycleListModelUnitTests: XCTestCase {
     var projectId = 123
     var cycle1Name = "cycle1"
     var cycle2Name = "cycle2"
+    var testPlanId = 12
     var testCycleItemType = 36
```

Comment:
Yes I refactored it to be a constant that has a inline comment explaining the numbers. It is the Jama system ID for test cycles.